misc/systemd/cf-consumer.service.in

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=CFEngine Enterprise Report Consumer
+PartOf=cfengine3.service
+After=syslog.target
+After=cf-postgres.service
+After=cf-redis-server.service
+ConditionPathExists=@bindir@/cf-consumer
+BindsTo=cf-redis-server.service
+BindsTo=cf-postgres.service
+
+[Service]
+Type=simple
+ExecStart=@bindir@/cf-consumer --no-fork --verbose
```

Comment:
`@bindir@` is not a replacement word (you can see which ones are by the `AC_SUBST` directives in `configure.ac`). I don't think it should be added because it is a direct descendant from the workdir, and if you wanted to move the bindir by itself, what you'd do is to move all the others (masterdir, piddir, inputdir, etc) except the workdir.

IOW: `s:@bindir@:@workdir@/bin:`